### PR TITLE
unbound output

### PR DESCRIPTION
**Pull request by zhangxiang1993**

### CODE REVIEW

**Comment by ryanlai2** [comment]
Should the code provide examples both unbound  and bound outputs?

**Comment by zhangxiang1993** [comment]
I think we should pick one, bound or unbound, to make the sample focus just on how to use batch.

---
In reply to: [590580532](https://github.com/microsoft/Windows-Machine-Learning/pull/296#issuecomment-590580532) [](ancestors = 590580532)

**Comment by zhangxiang1993** [comment]
Added the binding output code back as comments


---
In reply to: [590606427](https://github.com/microsoft/Windows-Machine-Learning/pull/296#issuecomment-590606427) [](ancestors = 590606427,590580532)

File: Samples/BatchSupport/BatchSupport/main.cpp [hunk 1]
```diff
@@ -78,6 +81,13 @@ int main(int argc, char *argv[]) {
   printf("model run took %d ticks\n", ticks);
 
   // Print Results
+
+  // conment three lines below if bind output
```
Comment (ryanlai2) [inline Samples/BatchSupport/BatchSupport/main.cpp]: Typo : Comment out three lines below when using bound output.

File: Samples/BatchSupport/BatchSupport/main.cpp [hunk 2]
```diff
@@ -78,6 +81,13 @@ int main(int argc, char *argv[]) {
...
+  auto outputValue = results.Outputs().Lookup(L"softmaxout_1").as<TensorFloat>(); // Get outputs by output name
+  auto outputShape = outputValue.Shape();
+  printf("output dimensions [%d, %d, %d, %d]\n", outputShape.GetAt(0), outputShape.GetAt(1), outputShape.GetAt(2), outputShape.GetAt(3));
+  // conment three lines above if bind output
```
Comment (ryanlai2) [inline Samples/BatchSupport/BatchSupport/main.cpp]: Typo : Comment out three lines above when using bound output.

**Pull request merged**